src/app/beer_garden/db/mongo/util.py

```diff
@@ -376,7 +321,7 @@ def check_indexes(document_class):
         try:
             document_class.ensure_indexes()
             logger.warning("%s indexes rebuilt successfully", document_class.__name__)
-        except OperationFailure:
+        except IndexOperationError:
```

Comment:
You don't want to change this one.  `ensure_indexes()` isn't going to raise the new `IndexOperationError` that you've introduced.